_posts/2023-01-23-there-is-no-cloud.md

```diff
@@ -0,0 +1,292 @@
+# There Is No Cloud, It's Just Someone Else's Computer
```

Comment:
The topic doesn't reveal this blog is about data encryption in cloud environments.

This is perhaps too long, but maybe:
"Encrypt Because The Cloud Is Just Someone Else's Computer"